public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -77,30 +77,32 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
           // Todo -- add this to api maybe
           let didNotAnswerCount  = this.survey.participantCount;
           q.selectedAnalysis = q.genderAnalysis;
-          for (const a of q.countAnalysis.results) {
-            if (q.subQuestions) {
-              const matchedQuestionConcepts: QuestionConcept[] = q.subQuestions.filter(
-                w => w.conceptName.toLowerCase() === a.stratum4.toLowerCase());
-              if (matchedQuestionConcepts.length === 0) {
-                q.subQuestions.push({
-                  conceptId: a.stratum3,
-                  conceptName: a.stratum4,
-                  domainId: 'ppi',
-                  conceptCode: '',
-                  countValue: a.countValue,
-                  prevalence: a.prevalence,
-                  countAnalysis: this.makeAnalysis(q.countAnalysis),
-                  genderAnalysis: this.makeAnalysis(q.genderAnalysis),
-                  ageAnalysis: this.makeAnalysis(q.ageAnalysis),
-                  genderIdentityAnalysis: this.makeAnalysis(q.genderIdentityAnalysis),
-                  subQuestions: null
-                });
+          if (q.countAnalysis && q.countAnalysis.results) {
```

Comment:
What does having a null count analysis mean? is this expected? Could you add a comment here?